### PR TITLE
Feature/refactor graphs components

### PR DESCRIPTION
**Pull request by cazapatamar**
Changes on RenderGraph and GraphLoader components to receive props

### CODE REVIEW

File: src/charts/RenderGraph.jsx [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
- * @param {any} data Graph data, it can be null (data hasn't loaded), false (data not available)
- *  or an Object with the data.
+ * @param {any} data Graph data, it can be null (data hasn't loaded),
+ *  false (data not available) or an Object with the data.
  * @param {string} labelX axis X label
  * @param {string} labelY axis Y label
  * @param {string} graph graph type
  * @param {string} graphTitle graph title
  * @param {array} colors colors to sort elements inside the graph
  */
```
Comment (erikasv) [inline src/charts/RenderGraph.jsx:17]: please add documentation for the rest of the properties

Comment (cazapatamar) [reply src/charts/RenderGraph.jsx:21]: Done

File: src/charts/GraphLoader.jsx [hunk 2]
```diff
@@ -12,27 +12,53 @@ import DotInfo from './DotInfo';
 
 ReactChartkick.addAdapter(Chart);
 
-const GraphLoader = (
-  {
-    graphType, data, graphTitle, labelX, labelY, width, height,
-    elementOnClick, colors, activeBiome, showOnlyTitle,
+const GraphLoader = (props) => {
+  const {
```
Comment (alegnaaived) [inline src/charts/GraphLoader.jsx:16]: Could be removed "const" and be use only {}?

Comment (alegnaaived) [reply src/charts/GraphLoader.jsx:16]: I tested something like this and works too:
image
image

**Pull request merged**